website/docs/community/resources/viewpoint.md

```diff
@@ -68,3 +68,11 @@ Here’s one example of an automated workflow:
 - code is deployed.
 
 Workflows like this should be built to execute with a single command.
+
+## The orchestration and control flow of analytics pipelines are configured declaratively 
+
+A data professional should maximize the time spent working with data. The configuration of where, how frequently, and in what order the transformations are executed should therefore be declartive, in order to minimize time operationalizing and maintaining workflows.
```

Comment:
I'm not sure I'm capturing the intent appropriately so I'm open to chagnes, but here's my suggestion to make this sentence active:
```suggestion
A data professional should maximize their time spent working with data by minimizing time operationalizing and maintaining workflows. With this efficiency in mind, you should declaratively configure where, how frequently, and in what order you execute transformations.
```